components/generic/RightsStatementButton.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <SmartLink
+    v-if="rightsStatementIsUrl"
+    :destination="rightsStatement"
+    class="rights-statement-button attribution mb-3 mb-lg-0"
+    data-qa="rights statement"
+  >
+    <RightsStatement
+      :rights-statement-url="rightsStatement"
+    />
+  </SmartLink>
+</template>
+
+<script>
+  import RightsStatement from '../../components/generic/RightsStatement';
+  import SmartLink from '../../components/generic/SmartLink';
+
+  export default {
+    components: {
+      RightsStatement,
+      SmartLink
+    },
+    props: {
+      rightsStatementIsUrl: {
```

Comment:
I don't think this really makes sense as a prop as it is conditional for rendering the entire template. This should instead be left to the parent view to determine whether or not to even render the RightsStatementButton component. Remove this prop.

Reply:
true, I have removed it